src/renderer/components/layout/setting-layout.tsx

```diff
@@ -104,11 +104,19 @@ export class SettingLayout extends React.Component<SettingLayoutProps> {
           <div className="toolsRegion">
             {
               this.props.provideBackButtonNavigation && (
-                <div className="fixedTools">
-                  <div className="closeBtn" role="button" aria-label="Close" onClick={back}>
-                    <Icon material="close" />
+                <div className="fixed top-[60px]">
+                  <div
+                    className="w-[35px] h-[35px] grid place-items-center pointer border-2 rounded-full border-solid border-textDimmed hover:bg-[#72767d25] active:translate-y-px"
```

Comment:
I would better to move all specific `tailwind` classes to style file with `@apply` instruction.

_Reason_: if we decide to get rid of tailwind in the future it would be much easier update css-files instead of cleaning it up from react-components.